package.json

```diff
@@ -22,8 +22,8 @@
 		"test": "grunt test",
 		"ci": "grunt ci"
 	},
-	"dependencies": {
-		"jquery": "3.0.0"
+	"peerDependencies": {
+		"jquery": ">3.0.0"
```

Comment:
True, and I wonder if I should also limit to `<4.0.0` since we'll plan on a new round of removals then?